### PR TITLE
Fixing Chap 20 flaky tests

### PR DESCRIPTION
**Pull request by Keboo**
This has the side benefit of also removing WebClient. Need to review book content to see if there would need to be chapter changes to go with this.

### CODE REVIEW

**Comment by MarkMichaelis** [comment]
Wow @Keboo, nice job!

**Review by MarkMichaelis — APPROVED** [review]
Wow @Keboo, nice job

File: src/Chapter20.Tests/Listing20.03.Tests.cs [hunk 1]
```diff
@@ -9,11 +9,13 @@ public class ProgramTests : BaseProgramTests
     [ClassInitialize]
     public static void ClassInitialize(TestContext _)
     {
+        Program.HttpClient = GetMockedHttpClient();
         ProgramWrapper = new ProgramWrapper(
             (args) =>
                 Task.Run(() => Program.Main(args)));
     }
 
+
```
Comment (BenjaminMichaelis) [inline src/Chapter20.Tests/Listing20.03.Tests.cs]: nit: spacing

**Review by BenjaminMichaelis — APPROVED** [review]

**Review by MarkMichaelis — APPROVED** [review]

File: src/Chapter20.Tests/Listing20.03.Tests.cs (hunk 1 above)
Comment (BenjaminMichaelis) [inline src/Chapter20.Tests/Listing20.03.Tests.cs]:
```suggestion
```

**Pull request merged**